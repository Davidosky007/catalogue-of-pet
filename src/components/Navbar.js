import React from 'react';
import { Link } from '@reach/router';

const Navbar = () => (
  <header className="main_header">
    <Link to="/">PetFinder</Link>
  </header>
);
export default Navbar;
