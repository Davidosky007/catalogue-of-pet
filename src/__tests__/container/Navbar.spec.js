import React from 'react';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import Navbar from '../../components/Navbar';

test('should render Details components properly ', () => {
  const details = renderer.create(<Navbar />);
  const final = details.toJSON();
  expect(final).toMatchSnapshot();
});
