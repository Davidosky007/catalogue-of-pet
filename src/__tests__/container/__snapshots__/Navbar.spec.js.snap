// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Details components properly  1`] = `
<header
  className="main_header"
>
  <a
    aria-current="page"
    href="/"
    onClick={[Function]}
  >
    PetFinder
  </a>
</header>
`;
